OracleLinuxDevelopers/oraclelinux8/ocne-tools/2.0/help.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+# Copyright (c) 2024 Oracle and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+echo "OCNE/OKE Tools"
+
+echo "oci-cli version: $(oci --version)"
+helm version --template='helm version: {{.Version}}'
+echo ''
+echo "kubectl versions: "
+kubectl version --client
```

Comment:
I suggest storing each version in a variable first, then printing them, ie. something like:

```bash
OCI_CLI_VERSION="$(oci --version)"
HELM_VERSION="$(helm version)"
KUBECTL_VERSION="$(kubectl version --client)"

echo "oci-cli version: $OCI_CLI_VERSION"
echo "helm version: $HELM_VERSION"
echo "kubectl version: $KUBECTL_VERSION"
```

It would be better to use `printf` but I can't recall the exact syntax off the top of my head. :)